projects/cashmere-examples/src/lib/drawer-filter-menu/drawer-filter-menu-example.html

```diff
@@ -0,0 +1,30 @@
+<div class="drawer-container">
+    <hc-drawer-container>
+        <hc-menu-drawer align="left" #leftDrawer>
+            <div hcDrawerToolbar>Filter Sets</div>
+            <hc-drawer-item>Clinical Program</hc-drawer-item>
+            <hc-drawer-item>Care Process Family</hc-drawer-item>
+            <hc-drawer-item>Care Process</hc-drawer-item>
+            <hc-drawer-item>Sub-Care Process</hc-drawer-item>
+            <hc-drawer-item>MS-DRG</hc-drawer-item>
+            <hc-drawer-item>Discharge Status</hc-drawer-item>
+            <hc-drawer-item>ER Visit</hc-drawer-item>
+            <hc-drawer-item>Payer</hc-drawer-item>
+            <hc-drawer-item>Age</hc-drawer-item>
+        </hc-menu-drawer>
+
+        <hc-drawer align="right" #rightDrawer>
+            Right drawer content
+        </hc-drawer>
+
+
+        <div style="height:200px;">
+            <button hc-button buttonStyle="primary" (click)="leftDrawer.toggle()" class="demo-button">
+                Toggle Left Drawer
+            </button>
+            <button hc-button buttonStyle="primary" (click)="rightDrawer.toggle()" class="demo-button">
```

Comment:
I might just be missing something, but what are we demonstrating with the `rightDrawer` in this example?  The left drawer looks great, though!  Excited to see an example of a filter menu using Cashmere components.

Reply:
I was wanting to show that hc-menu-drawer can be mixed with a hc-drawer. I wanted to show that hc-menu-drawer is nothing more than an extension of hc-drawer.